docs/get-invited-and-onboard-to-techpass.md

```diff
@@ -1,139 +1,100 @@
-# Get invited and onboard to TechPass
+# Get invited and onboard to TechPass (vendors and offshore users)
 
-This article guides you how to request for a TechPass account and onboard to it. 
+This guide explains how vendors and offshore users can request a TechPass account and complete onboarding.
 
-?>- Alternatively, public officers with a non-SE GSIB device can access the [**TechBiz Portal**](https://portal.techbiz.suite.gov.sg) to request for TechPass and SEED provisioning, which is optional. <br>- For more information, see [**TechBiz documentation**](https://docs.developer.tech.gov.sg/docs/techbiz-documentation/).
+?> For offshore users, SEED provisioning is **mandatory** unless you belong to an offshore development centre (ODC).
 
+---
 
-## Audience
+## Audience  
+Vendors and offshore users who require a TechPass account.
 
-Users who require a TechPass account but do not have a WOG account or a non-SE GSIB device.
+---
 
-## Prerequisites
+## Prerequisites  
 
-You need the following to get invited to TechPass and complete the onboarding:
+Ensure you have:  
+- A **standard organisational email address** (not LiteMail).  
+- Received the **TechPass onboarding email** (valid for 30 days).  
```

Comment:
I find it weird that it's a prerequisite that user have to first receive the onboarding email, but down below we ask them to Request for a TechPass account first.